dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -276,6 +283,19 @@ export default defineComponent({
         this.postLkSGDataProcessed = true;
       }
     },
+
+    /**
+     * updates the list of certificates that were uploaded in the corresponding formfields on change
+     * @param componentName the name of the component as a key
```

Comment:
Name of the component? Isn't this more like display name of the documend? Or this this really filled with the name of the component in which it was used like "CreateLksgDataSet)?

Reply:
This is the name of the component as it is specified in the LksgDataModel. It is used here as an identifier of all the fields that can hold a referenced document and emit changes regarding these documents.

Reply:
ok, thanks for your explanation in the call